.repair-bot/repro.py

```
@@ -0,0 +1,88 @@
+        ),
+    ),
+    (
+        "modelopt incompatibility",
+        re.compile(r"incompatible with nvidia-modelopt", re.IGNORECASE),
+    ),
+]
+
+
+def run_capture(cmd):
+    """Run cmd, return (returncode, combined_output)."""
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)